sessions/backend-update-delete/demo-end/components/JokeList/index.js

```diff
@@ -0,0 +1,30 @@
+import Button from "../Button";
+import List from "../List";
+import ListItem from "../ListItem";
+import useSWR from "swr";
+import { useRouter } from "next/router";
+
+export default function JokeList() {
+  const router = useRouter();
+  const { data } = useSWR("/api/jokes");
+
+  if (!data) {
+    return <h1>Loading...</h1>;
+  }
```

Comment:
shouldn't we use the isLoading from useSWR here?

Reply:
We absolutely should.